index.html

```diff
@@ -3617,28 +3617,56 @@ <h2 id="sotd">Status of This Document</h2>
               value associated with the key.
             </p>
             <p>
-              Prose accompanying an interface with a <a class="dfnref" href="#dfn-value-iterator">value iterator</a>
-              <span class="rfc2119">MUST</span> define what the
-              list of <dfn id="dfn-values-to-iterate-over">values to iterate over</dfn> is,
-              unless the interface also
-              <a class="dfnref" href="#dfn-support-indexed-properties">supports indexed properties</a>,
-              in which case the values of the indexed properties are implicitly
-              iterated over.  Prose accompanying an interface with a
+              A <a class="dfnref" href="#dfn-value-iterator">value iterator</a>
+              <span class="rfc2119">MUST</span> only be declared on an interface
+              that <a class="dfnref" href="#dfn-support-indexed-properties">supports indexed properties</a>
+              and has an <a class="dfnref" href="#dfn-integer-type">integer-typed</a>
+              <a class="dfnref" href="#dfn-attribute">attribute</a> named “length”.
+              The value-type of the <a class="dfnref" href="#dfn-value-iterator">value iterator</a>
+              <span class="rfc2119">MUST</span> be the same as the type returned by
+              the <a class="dfnref" href="#dfn-indexed-property-getter">indexed property getter</a>.
+              A <a class="dfnref" href="#dfn-value-iterator">value iterator</a> is implicitly
+              defined to iterate over the object’s indexed properties.
+            </p>
+            <p>
+              A <a class="dfnref" href="#dfn-pair-iterator">pair iterator</a>
+              <span class="rfc2119">MUST NOT</span> be declared on an interface
+              that does <a class="dfnref" href="#dfn-support-indexed-properties">support indexed properties</a>.
```

Comment:
"that supports indexed properties" instead of "that does support indexed properties"?